lib/oli_web/live/projects/active_sections_table_model.ex

```diff
@@ -64,6 +75,20 @@ defmodule OliWeb.Projects.ActiveSectionsTableModel do
   def custom_render(assigns, section, %ColumnSpec{name: :base_project_id}),
     do: if(section.base_project_id == assigns.project.id, do: "Base Project", else: "Remixed")
 
+  def custom_render(_assigns, section, %ColumnSpec{name: :creator}) do
+    case Publishing.find_oldest_enrolled_instructor(section) do
```

Comment:
Executing DB calls within the rendering phase is generally not a good idea, as we have zero control over how many times these DB calls will be made.  Every time a user sorts a column, these DB calls will execute. 

Instead, I would recommend creating a specialized "find_oldest_instructor" that took as input either a list of all sections, or maybe the project_id.  That call can then be made one time in the mount of the LiveView, and then that data passed down into the table model.